src/betamax/configure.py

```diff
@@ -2,7 +2,6 @@
 
 
 class Configuration(object):
-
```

Comment:
This whitespace change was unintentional, however I believe it's a (minor) style improvement. I'm happy to undo it if requested.